<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QPushButton" name="clearTimeSerias">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>181</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Очистить временной ряд</string>
         </property>
        </widget>
        <widget class="QPushButton" name="makeTaylorSeries">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ряд Тейлора</string>
         </property>
        </widget>
        <widget class="QPushButton" name="makeLagrangePolynomial">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Лаграндж</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="taylorSpinBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
        <widget class="QPushButton" name="eraseLastTimeSeriesPoint">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>161</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Удалить последнюю точку</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="polynomialDegree">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
        <widget class="QPushButton" name="approximationLS">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>МНК</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="lsPolinomeDegree">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>100</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>30.000000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
        <widget class="QPushButton" name="makeTaylorSeries_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ряд Фурье</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="lsPolinomeDegree_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>130</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="plot" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
